### PR TITLE
COMMERCE-12455 - Automation for COMMERCE-12298 and COMMERCE-12225

### PR DESCRIPTION
**Pull request by jaydawu**
https://liferay.atlassian.net/browse/COMMERCE-12455
https://liferay.atlassian.net/browse/COMMERCE-12483

### CODE REVIEW

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:stop

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:stop

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:stop

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase [hunk 1]
```diff
@@ -0,0 +1,782 @@
...
+		}
+
+		task ("Then the ABS Sensor price is the base price(20.00)") {
+			AssertTextEquals.assertPartialText(
+				key_cartItemSku = "MIN93015",
+				locator1 = "CommerceAccelerators#MINI_CART_ITEM_PRICE_VALUE",
+				value1 = "$ 20.00");
+		}
+	}
+
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase:238]: Please move this test to CPCommercePriceLists.testcase and remove the existing test CPCommercePriceLists#ViewExpiredTierPriceEntry that is duplicating the same scenario

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase [hunk 2]
```diff
@@ -0,0 +1,782 @@
...
+	@description = "This is a test for COMMERCE-12473. As an admin, I can use the expiration date for the bulk pricing in a product's SKU/UOM combination"
+	@priority = 4
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase:240]:
```suggestion
	@priority = 3
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase [hunk 3]
```diff
@@ -0,0 +1,782 @@
...
+		task ("Then the ABS Sensor price is the base price(20.00)") {
+			AssertTextEquals.assertPartialText(
+				key_cartItemSku = "MIN93015",
+				locator1 = "CommerceAccelerators#MINI_CART_ITEM_PRICE_VALUE",
+				value1 = "$ 20.00");
+		}
+	}
+
+	@description = "This is a test for COMMERCE-12474. As an admin, I can use the expiration date for the tiered pricing in a product's SKU"
+	@priority = 4
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase:373]:
```suggestion
	@priority = 3
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase [hunk 4]
```diff
@@ -0,0 +1,782 @@
...
+			AssertTextEquals.assertPartialText(
+				key_cartItemSku = "MIN93015",
+				locator1 = "CommerceAccelerators#MINI_CART_ITEM_PRICE_VALUE",
+				value1 = "$ 20.00");
+		}
+	}
+
+	@description = "This is a test for COMMERCE-12473. As an admin, I can use the expiration date for the bulk pricing in a product's SKU/UOM combination"
+	@priority = 4
+	test CanUseTheExpirationDateForBulkPricingUOM {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase:241]:
```suggestion
	test CanAssertExpirationDateForBulkPricingUOM {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase [hunk 5]
```diff
@@ -0,0 +1,782 @@
...
+
+			AssertElementPresent(
+				key_text = "Request a Quote",
+				locator1 = "Button#ANY");
+		}
+	}
+
+	@description = "This is a test for COMMERCE-12475. As an admin, I can use the expiration date for the bulk pricing in a product's SKU"
+	@priority = 4
+	test CanUseTheExpirationDateForBulkPricing {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase:137]:
```suggestion
	test CanAssertExpirationDateForBulkPricing {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase [hunk 6]
```diff
@@ -0,0 +1,782 @@
...
+			AssertTextEquals.assertPartialText(
+				key_cartItemSku = "MIN93015",
+				locator1 = "CommerceAccelerators#MINI_CART_ITEM_PRICE_VALUE",
+				value1 = "$ 20.00");
+		}
+	}
+
+	@description = "This is a test for COMMERCE-12474. As an admin, I can use the expiration date for the tiered pricing in a product's SKU"
+	@priority = 4
+	test CanUseTheExpirationDateForTieredPricing {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase:374]:
```suggestion
	test CanAssertExpirationDateForTieredPricing {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase [hunk 7]
```diff
@@ -0,0 +1,782 @@
...
+			}
+		}
+
+		task ("Then the ABS Sensor price is the base price(20.00)") {
+			AssertTextEquals.assertPartialText(
+				key_cartItemSku = "MIN93015",
+				locator1 = "CommerceAccelerators#MINI_CART_ITEM_PRICE_VALUE",
+				value1 = "$ 20.00");
+		}
+	}
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase:474]: Please move this in CPCommercePriceLists

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase [hunk 8]
```diff
@@ -0,0 +1,782 @@
...
+
+	@description = "This is a test for COMMERCE-12472. As an admin, I can use the expiration date for the tiered pricing in a product's SKU/UOM combination"
+	@priority = 4
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase:477]:
```suggestion
	@priority = 3
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase [hunk 9]
```diff
@@ -0,0 +1,782 @@
...
+	test CanUseTheExpirationDateForTieredPricingUOM {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase:478]:
```suggestion
	test CanAssertExpirationDateForTieredPricingUOM {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase [hunk 10]
```diff
@@ -0,0 +1,782 @@
...
+		task ("Then the ABS Sensor price is the base price(20.00)") {
+			AssertTextEquals.assertPartialText(
+				key_cartItemSku = "MIN93015",
+				locator1 = "CommerceAccelerators#MINI_CART_ITEM_PRICE_VALUE",
+				value1 = "$ 20.00");
+		}
+	}
+
+	@description = "This is a test for COMMERCE-12471. As an admin, I can view the UOM information for the SKU from the Price Lists and Promotion admin panels"
+	@priority = 4
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase:610]:
```suggestion
	@priority = 5
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase [hunk 11]
```diff
@@ -0,0 +1,782 @@
...
+	test CanViewUOMInformationFromPriceListsAndPromotions {
+		property portal.acceptance = "false";
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase:612]:
```suggestion
		property portal.acceptance = "true";
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase [hunk 12]
```diff
@@ -0,0 +1,782 @@
...
+
+				Click(locator1 = "Icon#CLOSE");
+
+				SelectFrame.selectFrameTop();
+			}
+		}
+	}
+
+	@description = "This is a test for COMMERCE-12470. As an admin, I can view the UOM information for the SKU from the SKU's price tab"
+	@priority = 4
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase:706]:
```suggestion
	@priority = 5
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase [hunk 13]
```diff
@@ -0,0 +1,782 @@
...
+	test CanViewUOMInformationFromSKU {
+		property portal.acceptance = "false";
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasurePricing.testcase:708]:
```suggestion
		property portal.acceptance = "true";
```

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]
@jaydawu please have a look at the comments I left, thanks!

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:forward

**Pull request closed without merging**